gufe/tokenization.py

```diff
@@ -157,6 +216,10 @@ def _gufe_tokenize(self):
         return tokenize(self)
         # return normalize(self.to_keyed_dict(include_defaults=False))
 
+    @classmethod
+    def serialization_migration(cls, old_dict, version):
+        return old_dict
```

Comment:
can this get documented?  From this I wouldn't know how to implement this in child classes...